app/src/main/java/com/infomaniak/mail/ui/main/search/SearchFragment.kt

```diff
@@ -239,21 +229,30 @@ class SearchFragment : Fragment() {
         }
     }
 
-    private fun setMessagesUi() {
-        binding.mailRecyclerView.apply {
+    private fun setMessagesUi() = with(binding) {
+
+        mailRecyclerView.adapter = threadListAdapter.apply {
```

Comment:
Why removing the adapter from ` mailRecyclerView.apply {}` ?

Reply:
Because it's an `apply` in an `apply` in a `with`, so it's unreadable.